src/server/controllers/index.js

```diff
@@ -58,7 +58,31 @@ function filterResponseFields(req, data, options) {
 }
 
 function filterResponseFieldsAll(req, arr, options) {
-  return arr.map(data => {
+  if (req.locals && req.locals.graphData) {
+    const combined = req.locals.graphData.map(i => {
+      const datum = arr.find(d => d[options.graphMatchingKey] === i.id);
+      if (datum) return Object.assign(i, datum.dataValues);
+      return i;
+    });
+    arr = combined;
+  }
+
+  if (options.thresholdKey && options.threshold) {
+    arr.sort((a, b) => {
+      const x = a[options.thresholdKey];
+      const y = b[options.thresholdKey];
+      return x < y ? -1 : x > y ? 1 : 0;
```

Comment:
Think you can write something like `return x-y` here and it would do the same :-)

Reply:
.. but also I'm wondering if it can already be sorted in the GraphQL query, not sure but maybe worth checking?